modules/accredited_representative_portal/app/models/accredited_representative_portal/representative_user.rb

```diff
@@ -7,24 +7,30 @@ class RepresentativeUser < Common::RedisStore
     redis_key :uuid
 
     # in alphabetical order
-    attribute :authn_context
-    attribute :email
-    attribute :fingerprint
-    attribute :first_name
-    attribute :icn
-    attribute :idme_uuid
-    attribute :last_name
-    attribute :last_signed_in
-    attribute :loa
-    attribute :logingov_uuid
-    attribute :ogc_registration_number
-    attribute :poa_codes
-    attribute :sign_in
-    attribute :uuid
+    attribute :authn_context, String
+    attribute :email, String
+    attribute :fingerprint, String
+    attribute :first_name, String
+    attribute :icn, String
+    attribute :idme_uuid, String
+    attribute :last_name, String
+    attribute :last_signed_in, Common::UTCTime
+    attribute :loa, String
+    attribute :logingov_uuid, String
+    attribute :sign_in, Hash
+    attribute :user_account_uuid, String
+    attribute :uuid, String
+
     alias_attribute :mhv_icn, :icn
 
     validates :uuid, :email, :first_name, :last_name, :icn, presence: true
 
+    def user_account
+      return unless user_account_uuid
```

Comment:
I'd just add this to the `validates` list for `presence: true` and then not bother with this early return. The `user_account` will always exist